### PR TITLE
[chore](regression) add smaller tpcds regression cases

### PR DESCRIPTION
**Pull request by ByteYue**
# Proposed changes

Issue Number: close #xxx
Add two new smaller size tpcds regression cases(each about 200M).
## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.6 seconds
 load time: 435 seconds
 storage size: 17172994779 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221125051504_clickbench_pr_52750.html

File: regression-test/data/tpcds_sf0.1_p1/sql/q01.out [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- This file is automatically generated. You should know what you did if you want to edit this
+-- !q01 --
+
```
Comment (Gabriel39) [inline regression-test/data/tpcds_sf0.1_p1/sql/q01.out:3]: Is it empty? If so, I think this scale doesn't work for correctness validation

Comment (ByteYue) [reply regression-test/data/tpcds_sf0.1_p1/sql/q01.out:3]: > Is it empty? If so, I think this scale doesn't work for correctness validation

It's so strange. I'll go check if some tables of the generated tpcds data are empty.

Comment (ByteYue) [reply regression-test/data/tpcds_sf0.1_p1/sql/q01.out:3]: > > Is it empty? If so, I think this scale doesn't work for correctness validation
> 
> It's so strange. I'll go check if some tables of the generated tpcds data are empty.

Yes the tables are valid. It seems 200M is still too small, i'll check if 500M or more works.

**Comment by ByteYue** [comment]
It seems tpcds data size under 1GB is not robust enough to test. This pr would be closed.

**Pull request closed without merging**